feat: add ecograder tool (#2)
zatsit-oss/awesome-impact-tools: CODE_OF_CONDUCT.md

# Code of Conduct - **zatsit** awesome impact tools

## Our Pledge

> In the interest of fostering an open and welcoming environment, we as

Reviewer: ```suggestion
In the interest of fostering an open and welcoming environment, we, as
```

> contributors and maintainers pledge to make participation in our project and

Reviewer: ```suggestion
contributors and maintainers, pledge to make participation in our projects and
```

our community a harassment-free experience for everyone, regardless of age, body
size, disability, ethnicity, sex characteristics, gender identity and expression,
level of experience, education, socio-economic status, nationality, personal
appearance, race, religion, or sexual identity and orientation.

## Our Standards

Examples of behavior that contributes to a positive environment for our
community include:

* Demonstrating empathy and kindness toward other people
* Being respectful of differing opinions, viewpoints, and experiences
* Giving and gracefully accepting constructive feedback
* Accepting responsibility and apologizing to those affected by our mistakes,
  and learning from the experience
* Focusing on what is best not just for us as individuals, but for the
  overall community

Examples of unacceptable behavior include:

> * The use of sexualized language or imagery, and sexual attention or

Reviewer: ```suggestion
* Use of sexualized language or images or advances
```

  advances
* Trolling, insulting or derogatory comments, and personal or political attacks
* Public or private harassment

> * Publishing others' private information, such as a physical or email

Reviewer: ```suggestion
* Publishing other people's private information, such as a physical or email
```
